server/sonar-server/src/test/java/org/sonar/server/usergroups/ws/UpdateActionTest.java

```diff
@@ -201,8 +203,7 @@ public void name_too_long() throws Exception {
 
   @Test
   public void forbidden_name() throws Exception {
-    GroupDto existingGroup = groupDao.insert(dbSession, new GroupDto().setName("old-name").setDescription("Old Description"));
-    dbSession.commit();
+    GroupDto existingGroup = insertGroupInDefaultOrganization(newGroupDto().setName("old-name").setDescription("Old Description"));
 
     expectedException.expect(IllegalArgumentException.class);
```

Comment:
expectMessage is missing, the IAE could be `"Group 'new-name' already exists"` for all we know